ps2/test.py

```diff
@@ -15,15 +15,35 @@ def get_available_letters(letters_guessed):
 letters_guessed.append(string1)
 print(letters_guessed)
 '''
-def match_with_gaps(my_word, other_word):
+
```

Comment:
What is this file doing exactly?

Reply:
It's for testing code before adding it to hangman.py This program is not actually done until I complete hangman_with_hints function. I will clean up these files then.